app/api/dao/user.py

```diff
@@ -39,11 +39,11 @@ def create_user(data: Dict[str, str]):
             A tuple with two elements. The first element is a dictionary containing a key 'message' containing a string which indicates whether or not the user was created successfully. The second is the HTTP response code.
         """
 
-        name = data["name"]
```

Comment:
Also please use a different branch for each pr. You've used develop currently, this may cause issues in the future.

Reply:
This will raise a key error if name entered is not found that's why none is required here